templates/header.hbs

```diff
@@ -1,4 +1,5 @@
 <a class="skip-navigation" tabindex="1" href="#main-content">{{t 'skip_navigation' }}</a>
+<a class="skip-to-chat-widget" tabindex="1" href="#launcher" >{{t 'skip_to_chat'}}</a>
 
```

Comment:
The string is "Skip to Chat" https://github.com/zendesk/help_center/blob/main/config/locales/translations/help_center.en.yml#L6830

Reply:
I had added a string for `skip to Web Widget` before. Is that ok or you think it should be widget and not have web.

Reply:
For some reason the translation for `skip_to_web_widget` is not found, although the [translations](https://github.com/zendesk/help_center/blob/main/config/locales/translations/help_center.en.yml#L6841) were merged long time ago in HC

Reply:
I think you also need to add the string to the [I18nKeysMapping hash](https://github.com/zendesk/help_center/blob/960acd208ba0b79e2a75158083a8948f6ef982c5/app/presenters/hbs/helpers/global_helper.rb#L31) used by the `t` helper

Reply:
Thanks I made a PR https://github.com/zendesk/help_center/pull/28668